src/pages/home/Home.page.tsx

```diff
@@ -132,7 +139,7 @@ export default function Home() {
```

Comment:
Change - Consider adding PropTypes or TypeScript interfaces for the `Card` component to validate `data` prop structure. This practice enhances code reliability by ensuring that the component receives the expected data shape.